vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -50,15 +51,22 @@ var (
 	ctrNamespaceLabel   = flag.String("container-namespace-label", "namespace", `Label name to look for container names`)
 	ctrPodNameLabel     = flag.String("container-pod-name-label", "pod_name", `Label name to look for container names`)
 	ctrNameLabel        = flag.String("container-name-label", "name", `Label name to look for container names`)
+
+	oomBumpUpRatio = flag.Float64("oom-bump-up-ratio", 1.2, `Specifies how much memory will be added after observing OOM`)
+	oomMinBumpUpMb = flag.Float64("oom-min-bump-up",  100, `Specifies minimal increase of memory in MB after observing OOM`)
 )
 
+
 func main() {
 	klog.InitFlags(nil)
 	kube_flag.InitFlags()
 	klog.V(1).Infof("Vertical Pod Autoscaler %s Recommender", common.VerticalPodAutoscalerVersion)
 
 	config := createKubeConfig(float32(*kubeApiQps), int(*kubeApiBurst))
 
+	model.OOMBumpUpRatio = *oomBumpUpRatio
+	model.OOMMinBumpUp = *oomMinBumpUpMb * 1024 * 1024 // convert to bytes
```

Comment:
I don't think we need to make the change in this PR, but just flagging that setting these as global package variables in this way feels a bit off. I think the better approach would be to construct a model struct and pass these in as fields.